crates/bevy_gizmos/src/gizmos.rs

```diff
@@ -589,6 +629,52 @@ impl Drop for Circle2dBuilder<'_, '_> {
     }
 }
 
+/// A builder returned by [`Gizmos::arc_2d`].
+pub struct Arc2dBuilder<'a, 's> {
+    gizmos: &'a mut Gizmos<'s>,
+    position: Vec2,
+    center_angle: f32,
+    arc_angle: f32,
+    radius: f32,
+    color: Color,
+    segments: usize,
+}
+
+impl Arc2dBuilder<'_, '_> {
+    /// Set the number of line-segements for this arc.
+    pub fn segments(mut self, segments: usize) -> Self {
```

Comment:
It's surprising to me that this is the only modifier exposed. Why is this a modifier rather than a function argument above?

Reply:
This mimics the existing API for circles and spheres that I wrote.
My reasoning being that it's much less likely to need to be configured, and has a reasonable default that could be made configurable in the future.